docs/connect/index.md

```diff
@@ -59,13 +59,13 @@ applications and drivers may obtain connection properties in different formats.
 **Connection-string examples**
 
 A native PostgreSQL connection string.
-`postgresql://<username>@<clustername>.cratedb.net/crate`
+`postgresql://<username>:<password>@<clustername>.cratedb.net/`
 
 A connection string for SQLAlchemy or Apache Flink.
-`crate://<username>@<clustername>.cratedb.net/crate`
+`crate://<username>:<password>@<clustername>.cratedb.net/?ssl=true`
```

Comment:
Afaik this is SQL Alchemy specific and not necessarily works that way for Flink

Reply:
a) Ah right.
b) You are right. We need to discriminate here.

Thanks!

Reply:
7f16fd14d48f278 intends to improve the situation in this regard.